administration/src/bp-modules/stores/StoresImportController.tsx

```diff
@@ -54,7 +55,7 @@ const StoresImport = ({ fields }: StoreImportProps): ReactElement => {
       setAcceptingStores([])
     },
     onError: error => {
-      const { title } = getMessageFromApolloError(error)
+      const { title } = getMessageFromApolloError(error, t)
```

Comment:
:question: might be a dumb question, but I see several places where we create const { t } = useTranslation('errors') and only use it to pass as a param into getMessageFromApolloError() function.
so I wonder why const { t } is not created inside getMessageFromApolloError() ?

same with getQueryResult(), we pass a parameter there, but it is not used and only passed to getMessageFromApolloError().

Reply:
thats because `useTranslation` is a hook and you can't use them in regular functions but in react functional components, so we pass them from functional components to the function
https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-from-react-functions

Reply:
Ah, thanks for the clarification